test/controllers/HomeControllerSpec.scala

```diff
@@ -339,5 +328,108 @@ class HomeControllerSpec extends BaseSpec with CurrentTaxYear {
 
       verify(controller.messageFrontendService, times(1)).getUnreadMessageCount(any())
     }
+
+  }
+
+  "Calling serviceCallResponses" should {
+
+    "return TaxComponentsDisabled status where there is not a PertaxContext and Nino" in new LocalSetup {
+      override def allowLowConfidenceSA: Boolean = false
```

Comment:
allowLowConfidenceSA is always false in this file, it can be made concrete in LocalSetup

Reply:
DOne